nominal/core.py

```diff
@@ -1025,3 +1025,22 @@ def _logs_to_conjure(
         elif isinstance(log, tuple):
             ts, body = log
             yield Log(timestamp=_flexible_time_to_integral_nanoseconds(ts), body=body)._to_conjure()
+
+
+def poll_until_ingestion_completed(datasets: Iterable[Dataset], interval: timedelta = timedelta(seconds=1)) -> None:
```

Comment:
is this intended as a public method or only for internal use?

Reply:
i think both; the top level version is just a passthrough, but figured if you're in `nominal.core` you probably won't be calling into the top-level functions much? idk, not a biggie either way of course